python/pyspark/sql/functions.py

```diff
@@ -144,12 +144,6 @@ def _():
                'measured in radians.',
 }
 
-_functions_2_2 = {
-    'to_date': 'Converts a string date into a DateType using the (optionally) specified format.',
-    'to_timestamp': 'Converts a string timestamp into a timestamp type using the ' +
-                    '(optionally) specified format.',
-}
-
```

Comment:
actually, that's right - we don't need them  - not sure if these are left behind from before `format` parameter was added or something.